src/Core/Defines.h

```diff
@@ -74,6 +74,10 @@
 /// Minimum revision supporting OpenTelemetry
 #define DBMS_MIN_REVISION_WITH_OPENTELEMETRY 54442
 
+/// Minimum revision supporting task processing on cluster
+#define DBMS_MIN_REVISION_WITH_CLUSTER_PROCESSING 54443
```

Comment:
Btw, I don't see where this value is used ...

Reply:
Fixed.

Reply:
It is not like I wanted you to remove `DBMS_MIN_REVISION_WITH_CLUSTER_PROCESSING`.
At least, initiator should check revisions of remote nodes and throw an exception if remove server is too old.